src/angular/styles/_typography.scss

```diff
@@ -2237,116 +1972,63 @@ $panelBorderRadius: 2;
   width: 100%;
   overflow-x: hidden;
   overflow-y: auto;
-  background-color: $sbbColorWhite;
-  border: pxToRem($panelBorderWidthStandard) solid $sbbColorGrey;
-  border-radius: 0 0 pxToRem($panelBorderRadius) pxToRem($panelBorderRadius);
+  background-color: var(--sbb-color-background);
+  border: var(--sbb-border-width) solid var(--sbb-color-grey);
+  border-radius: 0 0 var(--sbb-border-radius) var(--sbb-border-radius);
   border-top: none;
-  transition: height 0.25s cubic-bezier(0.4, 0, 0.2, 1), opacity 0.25s cubic-bezier(0.4, 0, 0.2, 1);
-  color: $sbbColorGrey;
+  transition: height var(--sbb-transition-panel), opacity var(--sbb-transition-panel);
+  color: var(--sbb-color-grey);
 
   @include panelShadow();
 
+  :where(html.sbb-lean) & {
+    border-color: var(--sbb-color-iron);
+    color: var(--sbb-color-iron);
+    @include panelShadow($borderWidth: $panelBorderWidthLean);
+  }
+
   hr {
     margin: pxToRem(10) 0;
     padding: 0;
     border: none;
-    border-top: solid pxToRem(1) $sbbColorCloud;
-  }
-
-  .sbb-label {
-    color: $sbbColorMetal;
-    padding: pxToRem(5) pxToRem(14);
-
-    html:not(.sbb-lean) & {
-      @include mq($from: desktop4k) {
-        padding: pxToRem(5 * $scalingFactor4k) pxToRem(14 * $scalingFactor4k);
-      }
-
-      @include mq($from: desktop5k) {
-        padding: pxToRem(5 * $scalingFactor5k) pxToRem(14 * $scalingFactor5k);
-      }
-    }
+    border-top: var(--sbb-border-width-thin) solid var(--sbb-color-cloud);
 
-    html.sbb-lean & {
-      color: $sbbColorGranite; // Redefine color to be more specific than standard variant
-      padding: pxToRem(4) pxToRem(8);
-    }
-  }
-
-  html:not(.sbb-lean) & {
-    @include mq($from: desktop4k) {
-      border-width: pxToRem($panelBorderWidthStandard * $scalingFactor4k);
-      border-radius: 0 0 pxToRem($panelBorderRadius * $scalingFactor4k)
-        pxToRem($panelBorderRadius * $scalingFactor4k);
-      @include panelShadow($scalingFactor: $scalingFactor4k);
-    }
-
-    @include mq($from: desktop5k) {
-      border-width: pxToRem($panelBorderWidthStandard * $scalingFactor5k);
-      border-radius: 0 0 pxToRem($panelBorderRadius * $scalingFactor5k)
-        pxToRem($panelBorderRadius * $scalingFactor5k);
-      @include panelShadow($scalingFactor: $scalingFactor5k);
+    :where(html.sbb-lean) & {
+      margin: pxToRem(4) 0;
     }
   }
 
-  html.sbb-lean & {
-    border-width: pxToRem($panelBorderWidthLean);
-    border-color: $sbbColorIron;
-    color: $sbbColorIron;
-    @include panelShadow($borderWidth: $panelBorderWidthLean);
+  .sbb-label {
+    color: var(--sbb-color-metal);
+    padding: calc(#{pxToRem(5)} * var(--sbb-scaling-factor))
+      calc(#{pxToRem(14)} * var(--sbb-scaling-factor));
 
-    hr {
-      margin: pxToRem(4) 0;
+    :where(html.sbb-lean) & {
+      // TODO: Check if still necessary
```

Comment:
I checked it. It is still necssary because sbb-label in standard has another color than sbb-label standard inside panel. So we have no other choice than overwrite this property in lean
```suggestion
```